src/examples/assetIssuance.ts

```diff
@@ -0,0 +1,123 @@
+import { BigNumber } from '@polymeshassociation/polymesh-sdk';
+import { Asset } from '@polymeshassociation/polymesh-sdk/types';
+
+import { getClient } from '~/common/client';
+import { parseArgs } from '~/common/utils';
+
+enum SCRIPT_ACTIONS {
+  ISSUE = 'issue',
+  REDEEM = 'redeem',
+  GET_HOLDERS = 'getHolders',
+}
+
+type ScriptArgs = {
+  amount?: number;
+  ticker?: string;
+  action?: SCRIPT_ACTIONS;
+};
+
+/* 
+  This script showcases Asset Issuance related functionality.It:
+    - Issues new tokens
+    - Get all asset holders
+    - Redeems tokens
+
+  Usage: yarn run-example ./src/examples/assetIssuance.ts ticker=<TICKER> action=<ACTION> amount=<INTEGER>
+*/
+(async (): Promise<void> => {
+  /**
+   * Get Token holders
+   * @param asset - Asset to get holders for
+   **/
+  const getHolders = async (asset: Asset): Promise<void> => {
+    const { data } = await asset.assetHolders.get();
+
+    console.log(`Holders of ${asset.ticker}: \n`);
+
+    data.forEach(({ identity, balance }) => {
+      console.log(`- Identity: ${identity.did}, Balance: ${balance.toFormat()}`);
+    });
+  };
+
+  /**
+   * Issue tokens for Asset
```

Comment:
Normally I like TSDoc comments, but in this case I think they distract from the example.

Its information about the script, and not the actual usage of the SDK, so I would remove these comments from here.